packages/warriorjs-core/src/Space.js

```diff
@@ -87,6 +112,7 @@ class Space {
   }
 
   /**
+   * WARNING: Deprecared method
```

Comment:
Let's remove `.isCaptive()` instead of deprecating it as we are still in [initial development phase](https://semver.org/#spec-item-4).